### DISCUSSION TITLE
computer specs for fastest processing

### DISCUSSION THREAD

**Discussion by posinski1987** [openai/whisper#409] [General]
Hello,

We have tested the large model across multiple GPUs.  So far the A4000 (16GB) 2080ti (11GB) and 4090 (24GB).

We have not seen a significant improvement between the GPUs despite the varying specs.  These were all run in HP z4g4s with intel i7 7820x CPUs, 32GB RAM, and Samsung SSD Boot drives.  I would like to build a computer optimized for running transcriptions but am not sure what to optimize.

1. Would it be helpful to run this on a machine using pcie 5.0?
2. How much does CPU and RAM come into play when running this on a CUDA GPU?
3. How important is the storage speed of the windows OS disk?
4. Is the RTX 4090 the best desktop model GPU to use?
5. Are multi-GPU configurations possible?  If so is there any changes I would need to make to allow that?

Thank you

**Comment by shervinemami** [comment]
Hi, can you post a comparison of how fast whisper performed on your 3 different GPUs? 
I posted some speed tests on a fast CPU and a "GTX 1660 Super" GPU, it would be good to see just how well different GPUs perform: https://github.com/openai/whisper/discussions/391

**Comment by FurkanGozukara** [comment]
I have a performance video here can you compare?

https://www.youtube.com/watch?v=q8Q8CCDdSKo

in 10 minutes and in large model it can  process 23 min english speech

**Comment by mihadoo** [comment]
There are some whisper performance results on this page: 

[https://kb.fab-online.com/0110-fabsubtitler-server/00060-gpuusage/ ](https://kb.fab-online.com/0110-fabsubtitler-server/00060-gpuusage/ )

It seems that for a better GPU also a better CPU and Mainboard are required.
